src/aind_codeocean_api/codeocean.py

```diff
@@ -413,3 +413,29 @@ def get_result_file_download_url(
         url = f"{self.computation_url}/{computation_id}/{results_suffix}"
         response = requests.get(url, auth=(self.token, ""))
         return response
+
+    def update_permissions(
+        self, capsule_id: str, permissions: Dict
```

Comment:
Should be data_asset id instead of capsule_id. The API would be easier to implement if the input arguments were users, groups, and everyone instead of just permissions.